docs/architecture/platform-versions.md

```diff
@@ -103,6 +103,33 @@ Related observations:
     version vs. those with an earlier version tends to be about 5-10x
     more for our Android users than for those on Google Play at large.
 
+* Android [upstream's advice][android-doc-supporting-platforms] is:
+
+  > Generally, it’s a good practice to support about 90% of the active devices
+
+  with a link to the dashboard of [global Google Play figures][].
+  Based on the May 2019 figures quoted above, this advice roughly
+  corresponds to dropping an old platform at a threshold of about 1%
+  of our userbase.
+
+[android-doc-supporting-platforms]: https://developer.android.com/training/basics/supporting-devices/platforms
+[global Google Play figures]: https://developer.android.com/about/dashboards/
+
+* On the Play Store, when we upload a new app version that drops
+  support for a given OS version, the last APK that did support that
+  version remains available indefinitely (for better or for worse) for
+  users with that OS version to install.
```

Comment:
This isn't the case, though. We do have an allegedly-Jelly-Bean-compatible Zulip version visible on the Play Console, but it wasn't visible on Google Play itself from a Jelly Bean device.

Reply:
Oh, good to know -- thanks for checking on that!

That makes it kind of confusing what that page of the Play Console is saying, then. Do you have a theory?

Reply:
I thought of a plausible explanation, and put it in the doc (suitably caveated.)